torch/csrc/utils/tensor_numpy.cpp

```diff
@@ -110,7 +110,7 @@ PyObject* tensor_to_numpy(const at::Tensor& tensor) {
   // object of the ndarray to the tensor and disabling resizes on the storage.
   // This is not sufficient. For example, the tensor's storage may be changed
   // via Tensor.set_, which can free the underlying memory.
-  PyObject* py_tensor = THPVariable_Wrap(make_variable(tensor, false));
+  PyObject* py_tensor = THPVariable_Wrap(tensor);
```

Comment:
should we assert that this is a variable?

Reply:
Addressed in https://github.com/pytorch/pytorch/pull/22214.